Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -24,16 +24,22 @@ namespace Microsoft.Azure.CosmosRepository
     /// <inheritdoc/>
     internal class InMemoryRepository<TItem> : IRepository<TItem> where TItem : class, IItem
     {
+        internal int CurrentTs => (int)(DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0)).TotalSeconds;
```

Comment:
Again could be static constant in the lib

Reply:
Yeah that is a good idea! Will have a look now.